gnoland/website/views/funcs.html

```diff
@@ -115,18 +115,31 @@
 <script type="text/javascript" src="/static/js/marked.min.js"></script>
 <script type="text/javascript" src="/static/js/purify.min.js"></script>
 <script type="text/javascript">
-  function main() {
+  function main(options) {
+    const { sourceId = "source", targetId = "home", highlight = true } = options || {};
```

Comment:
just one doubt here on whether we should be using [object destructuring](https://caniuse.com/mdn-javascript_operators_destructuring), due to the fact it has less support in browsers than anything else we're using in JS. maybe keep it "dumber" and do this instead?

```suggestion
    const sourceId = options.sourceId || "source";
    const targetId = options.targetId || "home";
    const highlight = typeof options.sourceId !== 'undefined' ? options.sourceId : true;
```

Reply:
Yes you are right, we can do without it.

Reply:
5a828ef